vms/platformvm/validators/manager_test.go

```diff
@@ -1,452 +0,0 @@
-// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package validators
```

Comment:
q: no tests anymore?

Reply:
I believe we talked about how the property tests done in the `vm` covered all of these cases. No?